### PR TITLE
fix: update wallet existing err

### PR DESCRIPTION
**Pull request by thedoublejay**

### CODE REVIEW

**Review by monstrobishi — CHANGES_REQUESTED** [review]
It's not enough to update just the error text here, there are 2 separate errors one of which the text is correct for:
- The address is actually invalid because it does not belong to the user (by checking that they have the private key for this public address in their wallet already), this is already fine.
- They already added the address (Need to return a different error here)

**Comment by thedoublejay** [comment]
> It's not enough to update just the error text here, there are 2 separate errors one of which the text is correct for:
> - The address is actually invalid because it does not belong to the user (by checking that they have the private key for this public address in their wallet already), this is already fine.
> - They already added the address (Need to return a different error here)

I see. Ok will do the necessary updates. Thanks!

File: webapp/src/containers/WalletPage/components/ReceivePage/CreateNewAddressPage/index.tsx [hunk 1]
```diff
@@ -94,19 +100,22 @@ const CreateNewAddressPage: React.FunctionComponent<CreateNewAddressPageProps> =
...
+    }
+    if (await isValidAddress(value)) {
+      setAddressValidity(AddressValidity.ERROR_INVALID_ADDRESS_FORMAT);
+      return;
+    }
+    if ((await isAddressMine(value)) && isAddressAlreadyExists(value)) {
+      setAddressValidity(AddressValidity.ERROR_EXIST_OWN_WALLET);
+      return;
+    }
+    if (await hdWalletCheck(value)) {
```
Comment (monstrobishi) [inline webapp/src/containers/WalletPage/components/ReceivePage/CreateNewAddressPage/index.tsx:115]: I know it's WIP but don't do HD wallet check here it's commented for a reason

Comment (thedoublejay) [reply webapp/src/containers/WalletPage/components/ReceivePage/CreateNewAddressPage/index.tsx]: Yup, just finding the right method to check the address existing check. It's not included on the final PR.

File: webapp/src/translations/languages/en.json [hunk 2]
```diff
@@ -651,7 +651,9 @@
         "addReceiveAddress": "Add a receive address",
         "address": "Address",
         "enterAnAddress": "Enter an address in your account",
-        "invalidAddress": "Invalid address. Please check this belongs to you, the current address has been used."
+        "existingAddressOthers": "Invalid address. Please check this belongs to you, the current address has been used.",
```
Comment (monstrobishi) [inline webapp/src/translations/languages/en.json:654]: I think we can remove "the current address has been used". It doesn't mean anything in this error message.

File: webapp/src/translations/languages/en.json [hunk 3]
```diff
@@ -651,7 +651,9 @@
...
+        "existingAddress": "Address exist in wallet.",
```
Comment (monstrobishi) [inline webapp/src/translations/languages/en.json:655]: The English here is weird, maybe "Address already exists"

**Review by monstrobishi — CHANGES_REQUESTED** [review]

**Pull request merged**